packages/astro-uxds/_content/components/clock.md

```diff
@@ -13,7 +13,7 @@ theme: true
 
 # Clock
 
-Clock shows the current time and, optionally, Julian date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
+Clock shows the current time and optional date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
```

Comment:
May be some branch leak here from the clock change - that should be fine in this case though.

Reply:
I un-did this change? Sorry about that- not sure how it was back in there.

Reply:
magic 🎉